BookStoreProd/src/main/java/mate/academy/bookstoreprod/controller/AuthenticationController.java

```diff
@@ -19,12 +19,11 @@
 public class AuthenticationController {
     private final UserService userService;
 
-    @PostMapping("/register")
+    @PostMapping("/auth/registration")
```

Comment:
add `@RequestMapping("/auth")` under controller
```suggestion
    @PostMapping("/registration")
```